source/expressions.tex

```diff
@@ -1690,7 +1690,20 @@
 
 \begin{bnf}
 \nontermdef{lambda-specifiers}\br
-    \opt{decl-specifier-seq} \opt{noexcept-specifier} \opt{attribute-specifier-seq} \opt{trailing-return-type}
+    \opt{lambda-specifier-seq} \opt{noexcept-specifier} \opt{attribute-specifier-seq} \opt{trailing-return-type}
+\end{bnf}
+
+\begin{bnf}
+\nontermdef{lambda-specifier}\br
+    \keyword{consteval}\br
+    \keyword{constexpr}\br
+    \keyword{mutable}
+\end{bnf}
+
+\begin{bnf}
+\nontermdef{lambda-specifier-seq}\br
+    lambda-specifier\br
+    lambda-specifier lambda-specifier-seq\br
```

Comment:
Why a traling `\br` here but not above?

Reply:
There should not be a trailing `\br` here.

If you want to, I can fix that and force-push.

Reply:
fixed